src/algebra/lie/free.lean

```diff
@@ -29,23 +31,16 @@ currently have definitions of ideals, lattices of ideals, and quotients for
 `non_unital_non_assoc_semiring`, we construct our quotient using the low-level `quot` function on
 an inductively-defined relation.
 
-### Alternative construction (often used over field of characteristic zero, but not by us)
+### Alternative construction (needs PBW)
 
-By universality of the free Lie algebra, there is a natural morphism of Lie algebras:
-`ρ : free_lie_algebra R X →ₗ⁅R⁆ free_algebra R X`, where `free_algebra R X` is the free unital,
-associative algebra, regarded as a Lie algebra via the ring commutator.
+An alternative construction of the free Lie algebra on `X` is to start with the free unital
+associative algebra on `X`, regard it as a Lie algebra via the ring commutator, and take its
+smallest Lie subalgebra containing `X`. I.e.:
+`lie_subalgebra.lie_span R (free_algebra R X) (set.range (free_algebra.ι R))`.
 
-By uniqueness, the image of `ρ` is the Lie subalgebra generated by `X` in the free algebra, i.e.:
-`lie_subalgebra.lie_span R (free_algebra R X) (set.range (free_algebra.ι R))`. If `ρ` is injective
-we could thus take the Lie subalgebra generated by `X` in the free associative algebra as the
-definition of the free Lie algebra, and indeed some authors do this. This is valid at least over a
-field of characteristic zero but I do not believe `ρ` is injective for general `R`. Indeed the only
-proof I know that `ρ` is injective relies on using the injectivity of the map from a Lie algebra to
-its universal enveloping algebra, which does not hold in general. Furthermore, even when a Lie
-algebra does inject into its universal enveloping algebra, the only proofs I know of this fact
-use the Poincaré–Birkhoff–Witt theorem. Some related MathOverflow questions are
-[this one](https://mathoverflow.net/questions/61954/) and
-[this one](https://mathoverflow.net/questions/114701/).
```

Comment:
Is it worth keeping around these old links too?

Reply:
I don't think so. They were distantly-related to whether a certain map was always injective but the new question directly answers this.